### PR TITLE
[Docs] OR Review 2

### PR DESCRIPTION
**Pull request by luarss**
Target Changes
- Prioritize prebuilt > docker > local installation 
- Add OS support matrix 

Fixes https://github.com/The-OpenROAD-Project/OpenROAD/issues/688

### CODE REVIEW

**Comment by luarss** [comment]
@vvbandeira @dralabeing Current prototype for tabs https://openroad--4159.org.readthedocs.build/en/4159/main/src/README.html

image

**Comment by vvbandeira** [comment]
@luarss, DCO is missing. Could you sign the commits so we can proceed?

**Comment by luarss** [comment]
@vvbandeira Ready for review. The tabbed code table looks nicer now.
image

File: docs/user/Build.md [hunk 1]
```diff
@@ -9,9 +9,47 @@ git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD.git
 cd OpenROAD
 ```
 
-OpenROAD git submodules (cloned by the `--recursive` flag) are located in `src/`.
+OpenROAD git submodules (cloned by the `--recursive` flag) are located in `src/`. 
 
-The default build type is RELEASE to compile optimized code.
+```{note}
+There are 3 methods for building OpenROAD (in order of recommendation): prebuilt binaries, docker images, and finally local build.
```
Comment (vvbandeira) [inline docs/user/Build.md:15]:
```suggestion
There are three methods for building OpenROAD (in order of recommendation): prebuilt binaries, docker images, and finally, local build.  
```

File: docs/user/Build.md [hunk 2]
```diff
@@ -9,9 +9,47 @@ git clone --recursive https://github.com/The-OpenROAD-Project/OpenROAD.git
...
+OpenROAD git submodules (cloned by the `--recursive` flag) are located in `src/`. 
 
-The default build type is RELEASE to compile optimized code.
+```{note}
+There are 3 methods for building OpenROAD (in order of recommendation): prebuilt binaries, docker images, and finally local build.  
+```
+
+## Build with Prebuilt Binaries
+
+Courtesy of [Precision Innovations](https://precisioninno.com/), there are pre-built binaries
```
Comment (vvbandeira) [inline docs/user/Build.md:20]:
```suggestion
Courtesy of [Precision Innovations](https://precisioninno.com/), there are prebuilt binaries
```

**Comment by luarss** [comment]
@vvbandeira Can this be merged?

**Comment by luarss** [comment]
@maliberty May I ask if who is the active tool maintainer for `eqy`? Are there any docs that I can insert for this?

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Review by vvbandeira — APPROVED** [review]

**Comment by luarss** [comment]
@vvbandeira Ready to merge?

**Review by vvbandeira — COMMENTED** [review]
Debian 10 is EOL

**Pull request merged**